apps/explorer/src/pages/transaction-result/TransactionView.tsx

```diff
@@ -87,26 +52,14 @@ export function TransactionView({
                                 <TransactionSummary transaction={transaction} />
                             </div>
                         </TabPanel>
-                        {/* {hasEvents && (
-                        <TabPanel>
-                            <div className={styles.txevents}>
-                                <div className={styles.txeventsleft}>
-                                    {eventTitlesDisplay}
-                                </div>
-                                <div className={styles.txeventsright}>
-                                    {txEventDisplay}
-                                </div>
-                            </div>
-                        </TabPanel>
-                    )} */}
                         <TabPanel>
                             <Signatures transaction={transaction} />
                         </TabPanel>
                     </TabPanels>
                 </TabGroup>
             </div>
         ),
-        minSize: 35,
+        defaultSize: 35,
```

Comment:
either set the width for content, or we need `minSize` here so that we do not get squished content when the panel is too small